openpype/tools/sceneinventory/window.py

```diff
@@ -158,6 +171,42 @@ def _on_outdated_state_change(self):
             self._outdated_only_checkbox.isChecked()
         )
 
+    def _on_update_all(self):
```

Comment:
For now I would move this into view. For example the `_show_version_error_dialog` is not availabe here.

Reply:
Moved with https://github.com/pypeclub/OpenPype/pull/2805/commits/882a17b04a164c833de3bdbca06a1af5505eaf78